rum-react-navigation-codepush/README.md

```diff
@@ -33,3 +33,16 @@ To try this project for yourself you need to create a folder called `config` in
         "shopistApiHost": "shopist.io"
     }
     ```
+
+Moreover, since we have added the [`datadog-react-native-wizard`][1] you need to add the datadog-ci.json file in the project root containing:
```

Comment:
```suggestion
Since we have added the [`datadog-react-native-wizard`][1], you need to add the `datadog-ci.json` file in the project root that includes the following:
```